services.go

```diff
@@ -423,6 +424,7 @@ type SetSlackServiceOptions struct {
 	PushEvents             *bool   `url:"push_events,omitempty" json:"push_events,omitempty"`
 	WikiPageChannel        *string `url:"wiki_page_channel,omitempty" json:"wiki_page_channel,omitempty"`
 	WikiPageEvents         *bool   `url:"wiki_page_events,omitempty" json:"wiki_page_events,omitempty"`
+	BranchesToBeNotified   *string `url:"branches_to_be_notified,omitempty"` `json:"branches_to_be_notified,omitempty"`
```

Comment:
Please insert this at the same location as the other insert (so above `ConfidentialIssueChannel`).

Reply:
Ooops, done